simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -36,69 +36,78 @@
           </mesh>
         </geometry>
       </collision>
-     
-      <sensor type="multicamera" name="stereo_camera">
```

Comment:
Multicam wouldn’t publish correct frame id, and I couldn't find any way to do it.